tetragonpod/api/v1alpha1/client/register_crd.go

```diff
@@ -0,0 +1,43 @@
+package client
+
+import (
+	"context"
+
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/tetragonpod"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+var scopedLogger = logger.GetLogger()
+
+func RegisterCRD(client *apiextclientset.Clientset) error {
+	crd := GetCRD()
+	_, err := client.ApiextensionsV1().CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
```

Comment:
this will fail if the CRD already exists. we need to call Update() if the CRD already exists.

Reply:
oh thanks for pointing that out, I totally missed that. Working on it.

Reply:
I made changes, the current implementation is like this:

- The CRD is created only if it doesn't already exists.
- If the CRD already exists, no changes are made.

I am having a few questions regarding the update logic, like how would it know that CRD needs to be updated ? I went through the TracingPolicy CRD logic, how it is handling it, but have a few doubts. Let's have a little discussion on it in the meeting today.